src/__tests__/CompareResults/OverTimeResultsView.test.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jest/no-disabled-tests */
+import type { ReactElement } from 'react';
+
+import { loader } from '../../components/CompareResults/overTimeLoader';
+import OverTimeResultsView from '../../components/CompareResults/OverTimeResultsView';
+import { Strings } from '../../resources/Strings';
+import getTestData from '../utils/fixtures';
+import {
+  renderWithRouter,
+  screen,
+  FetchMockSandbox,
+} from '../utils/test-utils';
+
+function renderWithRoute(component: ReactElement) {
+  const { testCompareData, testData } = getTestData();
+  (window.fetch as FetchMockSandbox)
+    .get(
+      'begin:https://treeherder.mozilla.org/api/perfcompare/results/',
+      testCompareData,
+    )
+    .get('begin:https://treeherder.mozilla.org/api/project/', {
+      results: [testData[0]],
+    });
+
+  return renderWithRouter(component, {
+    route: '/compare-over-time-results/',
+    search: '?newRev=coconut&newRepo=try&framework=1&selectedTimeRange=86400',
+    loader,
+  });
+}
+
+jest.mock('../../utils/location');
+// const mockedGetLocationOrigin = getLocationOrigin as jest.Mock;
+
+describe('Results View', () => {
+  it('renders correctly', async () => {
+    renderWithRoute(
+      <OverTimeResultsView title={Strings.metaData.pageTitle.results} />,
+    );
+
+    expect(document.body).toMatchSnapshot();
+  });
+
+  it('The table should match snapshot and other elements should be present in the page', async () => {
+    renderWithRoute(
+      <OverTimeResultsView title={Strings.metaData.pageTitle.results} />,
+    );
+    expect(await screen.findByRole('table')).toMatchSnapshot();
+
+    const link = screen.getByRole('link', { name: /link to home/i });
+    expect(link).toBeInTheDocument();
```

Comment:
(optional) I would combine this with the previous testcase

also I'm not sure the 2 snapshots are useful, maybe one is enough?
```ts
// make sure everything is rendered (esp when we'll use deferred later)
await screen.findByRole('table')
// check the full page
expect(document.body).toMatchSnapshot();
```

The goal is to keep the test run fast enough; currently they're a bit slow, I think because we render a lot and we do a lot of snapshots -- but we should profile the tests to be sure :-) (later)

Reply:
Yeah that makes sense - Thanks!